support/cas-server-support-memcached-core/src/main/java/org/apereo/cas/memcached/kryo/CloseableKryoFactory.java

```diff
@@ -211,9 +208,15 @@ private static void registerNativeJdkComponentsWithKryo(final Kryo kryo) {
         kryo.register(Date.class, new DateSerializer(Date.class));
         kryo.register(Calendar.class, new GregorianCalendarSerializer());
         kryo.register(GregorianCalendar.class, new GregorianCalendarSerializer());
-        kryo.register(LocalDate.class, new JodaLocalDateSerializer());
```

Comment:
Please not that there was a bug here: the `JodaLocalDateSerializer` was registered for `java.time.LocalDate` instead of `org.joda.time.LocalDate`